README.md

```diff
@@ -65,5 +65,63 @@ To start the hello world workflow, run the following command:
 cadence --domain cadence-workshop wf start --tasklist tasklist --execution_timeout 10 --workflow_type HelloWorld --input '{"message":"Cadence"}'
 ```
 
-The workflow should start and you should see the workflow in the web UI:
+The workflow should start, and you should see the workflow in the web UI:
 http://localhost:8088/domains/cadence-workshop
+
+### Task 1: Postnord OrderProcessingWorkflow
+Implement the validatePayment() and shipProduct() activities for the OrderProcessingWorkflow. <br />
+The validatePayment() activity should check if the order amount is greater than 25 and return an error if it is, otherwise return true for valid payments. <br />
+The shipProduct() activity should verify that the order.Customer (representing the shipping address) is not empty; <br />
+if it is, return an error indicating that shipping failed, otherwise simulate shipping and return true for successful shipment. <br />
+To start the OrderProcessingWorkflow, run the following command:
+```bash
+cadence --domain cadence-workshop wf start --tasklist tasklist --execution_timeout 10 --workflow_type OrderProcessingWorkflow --input '{"id":"Order123", "customer": "Cadence", "amount": 20, "address": "Uber office"}'
+```
+
+### Task 2: Simulate Payment Failure in validatePayment() activity
+First, modify the processPaymentActivity to simulate a failure. This can be done by causing the activity to fail intermittently, such as by returning an error during the first few attempts, which will simulate a temporary issue like a network failure.
+<br />
+<br />
+Use the code below to simulate the failure:
+```go
+func processPaymentActivity(ctx context.Context, order Order) (string, error) {
+	// Simulate a failure
+	if time.Now().Unix()%2 == 0 { // Fail intermittently (every second attempt).
+		activity.GetLogger(ctx).Info("Temporary failure in payment processing")
+		return "", fmt.Errorf("temporary network issue")
+	}
+	
+	activity.GetLogger(ctx).Info("Payment processed successfully")
+	return "Payment successful", nil
+}
+```
+
+Ensure that the workflow fails intermittently and return an error.
+
+### Task 3: Add Retry Policy to validatePayment() activity
+Now, implement a retry policy to handle the temporary failures simulated in Task 2. <br /> 
+The retry policy should retry the validatePayment activity up to 3 times with exponential backoff. <br />
+Use the paymentRetryPolicy configuration below:
```

Comment:
I didn't follow this completely, but isn't `processPaymentActivity` has 50% chance of just working? If so, half of students may not even face with the issue. Maybe to drop this probability substantially?